### ISSUE TITLE
negative discriminator loss

### ISSUE THREAD

**Issue by dlnlpchenliyu** [taki0112/StarGAN-Tensorflow#12]
Hi~I meet a problem while i try to use your code to do painting style transfer.
I find that during training, the discriminator loss may be negative cause sometimes the generated image gets a lower loss than the real one from the discriminator which is not sufficiently trained.
Have you ever met the same problem?Will it fo favor to the final performance of our model?
What if I use a pre-trained discriminator?
Waiting for your reply.
